app/scripts/shell/dimSearchFilter.directive.js

```diff
@@ -332,6 +333,29 @@
 
         return (item.classType === value);
       },
+      glimmer: function(predicate, item) {
+        var boosts = [
+          1043138475, // -black-wax-idol
+          1772853454, // -blue-polyphage
+          3783295803, // -ether-seeds
+          3446457162  // -resupply-codes
+        ];
+        var supplies = [
+          269776572, // -house-banners
+          3632619276, // -silken-codex
+          2904517731, // -axiomatic-beads
+          1932910919 // -network-keys
+        ];
+        switch (predicate) {
+        case 'glimmerboost':
+          return boosts.includes(item.hash);
+        case 'glimmersupply':
+          return supplies.includes(item.hash);
+        case 'glimmeritem':
+          return boosts.concat(supplies).includes(item.hash);
```

Comment:
Might want to avoid the concat for every item by defining the three arrays outside the function.